test/Microsoft.AspNet.FileProviders.Combined.Tests/TestUtility/MockChangeToken.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.FileProviders.Combined.Tests.TestUtility
+{
+    internal class MockChangeToken : IChangeToken
+    {
+        private readonly List<Action<object>> _callsback = new List<Action<object>>();
+
+        public bool ActiveChangeCallbacks
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        public bool HasChanged
+        {
+            get; set;
+        }
+
+        public int NbOfCallback
```

Comment:
Do I make the same change to all readonly properties (eg ActiveChangeCallbacks) ?